pom.xml

```diff
@@ -60,7 +60,7 @@
         <dep.testng.version>7.5</dep.testng.version>
         <dep.lucene.version>8.11.3</dep.lucene.version>
         <dep.assertj-core.version>3.8.0</dep.assertj-core.version>
-        <dep.logback.version>1.2.13</dep.logback.version>
+        <dep.logback.version>1.3.15</dep.logback.version>
```

Comment:
Do we want to keep this here or just rely on airbase?

Reply:
I removed it